tools/generate-tables.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
```

Comment:
could you add a

set -euo pipefail

(or the closest portable equivalent) so that errors will be caught?


https://gist.github.com/vncsna/64825d5609c146e80de8b1fd623011ca